src/scss/layouts/_choosegamesection.scss

```
@@ -1,4 +1,8 @@
+
 .chooseGameSection {
   width: 70%;
   border: 1px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
```

Tutaj mogę zaporoponować małą zmianę związaną z konwencją w której mistrzem jest @BartoszDombrow. 
```suggestion
.chooseGameSection {
  display: flex;
  justify-content: center;
  align-items: center;
  width: 70%;
  border: 1px solid black;
}
```